guides/deployments/podman.md

```diff
@@ -137,69 +150,85 @@ Linux kernel doesn't support CVMs.
    ```yaml
    version: "0.2"
    workspace:
-       configure:
-           start:
-               policy: write
-       dev-urls:
+     configure:
+       start:
+         policy: write
+     dev-urls:
+       policy: write
+     specs:
+       aws-ec2-docker:
```

Comment:
whoops, I dunno how the stuff with ec2 and docker slipped in, but I don't think we need those?

Reply:
weird, I'll fix. sorry!

Reply:
ok, I'm not in the right headspace to fix this right now. will return to this in the AM.

Reply:
sounds good, thanks Katie!